.devcontainer/devcontainer.json

```diff
@@ -15,8 +15,9 @@
   "forwardPorts": [3000, 4000],
 
   // Use 'postCreateCommand' to run commands after the container is created.
-  "postCreateCommand": ".devcontainer/post-create.sh",
-  "waitFor": "postCreateCommand",
+  "postCreateCommand": "git config --global --add safe.directory ${containerWorkspaceFolder}",
+  "postStartCommand": ".devcontainer/post-create.sh",
```

Comment:
Why is the post-create script now being called on Start?

Reply:
Because `git config --global --add safe.directory ${containerWorkspaceFolder}` needs to be called before it’s run to prevent the `fatal: detected dubious ownership in repository` error.

Reply:
Yup, I did something similar in https://github.com/mastodon/mastodon/pull/23977
I just wasn't sure why the post-create script is now being called in a different part of the lifecyle. Should it be renamed if it's now called on start each time https://code.visualstudio.com/remote/advancedcontainers/start-processes?

Reply:
> it's now called on start each time

Oh wait. That  not what we want. I've corrected it. The `git config --global --add safe.directory ${containerWorkspaceFolder}` should be called with `onCreateCommand` not `postCreateCommand`, there's no reason to change `postCreateCommand`. Ive corrected that.